project/Build.scala

```diff
@@ -73,7 +73,9 @@ object DottyBuild extends Build {
       javaSource in Test := baseDirectory.value / "test",
       resourceDirectory in Compile := baseDirectory.value / "resources",
       unmanagedSourceDirectories in Compile := Seq((scalaSource in Compile).value),
+      unmanagedSourceDirectories in Compile += baseDirectory.value / "dottydoc" / "src",
```

Comment:
Have you tried making `dottydoc` a separate sbt project like `dotty-interfaces` and `dotty-bridge` ?